_includes/footer.html

```diff
@@ -1,42 +1,39 @@
-
 <footer class="page-footer">
   <div class="container">
     <div class="row">
-      <div class="col l6 s12">
+      <div class="col-md-6 col-xs-12">
         <h5 class="white-text">viperdev</h5>
-        <p class="grey-text text-lighten-4">The Leading Edge Way to Improve Code Quality</p>
-
+        <!-- Adding closing tags for increase in readability, HTMLLintBear asks to avoid closing p tag -->
```

Comment:
I filed https://github.com/coala/coala-bears/issues/1247 in the meantime IMO you should use `htmllint_ignore = optional_tag` in the coafile instead of ignoring it every single time